### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by Amanbek22**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
+          <Navbar basket={basket} />
+          <Main setBasket={setBasket} />
+        </Route>
+        <Route path="/contacts">
+          <Header />
+          <Navbar basket={basket} />
+          <Contacts />
+        </Route>
+        <PublicRoute path="/admin" auth={isAuth} component={() => <Admin setIsAuth={setIsAuth} />} />
+        {/* <Route path="/admin">
```
Comment (Esenushka) [inline src/App.js:38]: pls remove comments

File: src/App.js [hunk 2]
```diff
@@ -1,35 +1,59 @@
...
+          <Admin setIsAuth={setIsAuth} />
```
Comment (Esenushka) [inline src/App.js:39]: please remove comments

File: src/App.js (hunk 1 above)
Comment (Amanbek22) [reply src/App.js:38]: no. I need it later

**Review by Amanbek22 — COMMENTED** [review]
LGTM

**Review by Esenushka — APPROVED** [review]
LGTM

**Pull request merged**